homeassistant/components/sharkiq/vacuum.py

```diff
@@ -232,5 +265,6 @@ def extra_state_attributes(self) -> dict[str, Any]:
             ATTR_ERROR_MSG: self.sharkiq.error_text,
             ATTR_LOW_LIGHT: self.low_light,
             ATTR_RECHARGE_RESUME: self.recharge_resume,
+            ATTR_ROOMS: self.available_rooms,
```

Comment:
Do we need all rooms to be recorded by the recorder? I think we can exclude it, or what do you think?
See https://developers.home-assistant.io/docs/core/entity#excluding-state-attributes-from-recorder-history

Reply:
hmmm giving the user a place to go to see the valid values they can enter for room seems necessary.  it doesn't seem like something we need tracked over time, but being able to see the current value is kinda necesssary.  I'm happy to move this elsewhere if there's a better way.. but leaving for now.

Reply:
@jrlambs did you follow the link in @edenhaus' comment? He's not saying you should remove the new state attribute, he's saying you should add a line like this to the class definition to exclude the rooms attribute from state history.
```
_unrecorded_attributes = frozenset({ATTR_ROOMS})
```